make_figures.py

```diff
@@ -203,24 +231,74 @@ def make_timeplot(df_measure, df_prediction, countries=None):
-            yref='paper',
-            showarrow=False,
-            font_size=LABEL_FONT_SIZE - 6,
-            font_color="DarkSlateGray",
-            text="Drag handles below to change time window",
-            align="right")
+    # fig.add_annotation(
+    #         x=0.1,
+    #         y=0.95,
+    #         xref='paper',
+    #         yref='paper',
+    #         showarrow=False,
+    #         font_size=LABEL_FONT_SIZE,
+    #         text="Confirmed cases per Million")
+    # fig.add_annotation(
+    #         x=1,
+    #         y=-0.13,
+    #         xref='paper',
+    #         yref='paper',
+    #         showarrow=False,
+    #         font_size=LABEL_FONT_SIZE - 6,
+    #         font_color="DarkSlateGray",
+    #         text="Drag handles below to change time window",
+    #         align="right")
+    fatalities_annotation = [dict(x=0.1,
+                                 y=0.95,
+                                 xref='paper',
+                                 yref='paper',
+                                 showarrow=False,
+                                 font_size=LABEL_FONT_SIZE,
+                                 text="fatalities cases per Million"
+                                 )]
+    confirmed_annotation = [dict(x=0.1,
+                                 y=0.95,
+                                 xref='paper',
+                                 yref='paper',
+                                 showarrow=False,
+                                 font_size=LABEL_FONT_SIZE,
+                                 text="Confirmed cases per Million"
+                                 )]
+    drag_handle_annotation = [dict(x=1,
+                                   y=-0.13,
+                                   xref='paper',
+                                   yref='paper',
+                                   showarrow=False,
+                                   font_size=LABEL_FONT_SIZE - 6,
+                                   font_color="DarkSlateGray",
+                                   text="Drag handles below to change time window",
+                                   align="right")]
+
+    visibility_tag_inv = list(np.logical_not(visibility_tag))
+    fig.update_layout(
+    updatemenus=[
+        dict(
+            type="buttons",
+            direction="right",
+            active=0,
+            x=0.57,
+            y=1.2,
+            buttons=list([
+                dict(label="Active",
+                     method="update",
+                     args=[{"visible": visibility_tag},
+                           {"title": "Confirmed cases per Million",
+                            "annotations": confirmed_annotation + drag_handle_annotation}]),
+                dict(label="Fatalities",
+                     method="update",
+                     args=[{"visible": visibility_tag_inv},
+                           {"title": "Death cases per Million",
+                            "annotations": fatalities_annotation}]),
```

Comment:
maybe also drag handle annotation? Any good reason to remove it?

Reply:
I thought it will be needed for the active cases only, yes, you are right I should add drag handle annotation too